ql/src/java/org/apache/hadoop/hive/ql/ddl/table/storage/skewed/AlterTableSetSkewedLocationAnalyzer.java

```diff
@@ -20,7 +20,6 @@
 
 import java.net.URI;
 import java.net.URISyntaxException;
-import java.util.ArrayList;
```

Comment:
All changes in this file are out of scope of the PR.  Please revert.

Reply:
Removed.